Automattic/wp-calypso
# Stats: Move module-followers-page and module-followers

Another pass at #2314 - this branch moves the following stats components:

- `module-followers-page.jsx` -> `/stats-followers-page/index.jsx`
- `module-followers.jsx` -> `/stats-followers/index.jsx`

ES6 and general clean-up will be done in another branch.

__To Test__
- Load a stats insights page
- Confirm the Followers Component is shown.
- Click on the 'View All' Link at the bottom of the followers component
- Confirm the followers page is shown
